primedev/toolframework/itoolentity.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include "core/math/vector.h"
+
+class IClientTools // : public IBaseInterface
+{
+public:
+	virtual void sub_1805E4960() = 0;
+	virtual void sub_1805E4B10() = 0;
+	virtual void sub_1805E4C50() = 0;
+	virtual void sub_1805E5670() = 0;
+	virtual void sub_1805E66C0() = 0;
+	virtual void sub_1805E5910() = 0;
+	virtual void sub_1805E59A0() = 0;
+	virtual void sub_1805E6B10() = 0;
+	virtual void sub_1805E8580() = 0;
+	virtual void sub_1805E59D0() = 0;
+	virtual void sub_1805E57B0() = 0;
+	virtual void sub_1805E5860() = 0;
+	virtual void sub_1805E55A0() = 0;
+	virtual void sub_1805E49C0() = 0;
+	virtual void sub_1805E7580() = 0;
+	virtual void sub_1805E86A0() = 0;
+	virtual void sub_1805E69B0() = 0;
+	virtual void sub_1805E4D70() = 0; // IClientTools::DrawSprite
+	virtual void sub_1805E57A0() = 0;
+	virtual bool GetLocalPlayerEyePosition(Vector3& org, QAngle& ang, float& fov) = 0;
```

Comment:
I assume the remaining fields are just padding?

Reply:
cc @F1F7Y

Reply:
They are all just unknown functions, the names should match with ones that IDA generates